nearcore/src/config.rs

```diff
@@ -837,11 +842,11 @@ pub fn init_configs(
 
     if let Some(url) = download_config_url {
         download_config(&url.to_string(), &dir.join(CONFIG_FILENAME));
-        config = Config::from_file(&dir.join(CONFIG_FILENAME));
+        config = Config::from_file(&dir.join(CONFIG_FILENAME)).unwrap();
```

Comment:
I want to call-out this general pattern of pushing `.unwrap`s to the call site. It's a good approach, we should use it more:

* for the call-site, you still get the convenient "I don't care about failures" API
* but the failure points are more visible, and, if you decide in the future that after all you *do* care about this, refactoring would be easier. 
* implementation actually becomes easier, as you can use `?`, and you don't have to spend mental cycles to decide between `.unwrap` and `?`.